### DISCUSSION TITLE
Plot dvv of an specific pair of station

### DISCUSSION THREAD

**Discussion by AlvicePC** [ROBelgium/MSNoise#325] [Q&A]
Hi, i'm trying yo plot dvv of an specific pair of stationa but ir still ploting mean ando media of all pairs, in the Legend the pair that I want to graph appears but it does not appear on the graph I am using the command as follows

"msnoise plot dvv -p JA.ESTA_YELJA -f6 -c ZZ -M M0"
![17088620208434134174894894442604](https://github.com/ROBelgium/MSNoise/assets/86736263/41265c4a-a40a-4a90-8f7c-f90d6e030707)

**Comment by ThomasLecocq** [comment]
Did you check that the DTT folder contains data for that pair/filter?

**Reply by AlvicePC** [reply]
There is a folder for every pair into STACK and MWCS but no into DTT folder

**Reply by ThomasLecocq** [reply]
did you use the development version ? In that case, it's possible that the -p option is not yet implemented.
